ontology/yaml/resources/SAFETY/entity_types/FD.yaml

```diff
@@ -31,4 +31,7 @@ FD_SD:
   uses:
   - damper_percentage_sensor
   - damper_percentage_command
+  opt_uses:
+  - run_command
```

Comment:
what exactly is running?

Reply:
I realize now this is a mistake on my part. the run_command and _status is supposed to be for a motorized smoke detection damper. I will create a canonical now for that and revert this back sir.